_chapters/stats_and_probabilities.md

```diff
@@ -5,7 +5,7 @@ order: 8
 after: snake
 ---
 ### Liberal Draw Odds
-The first government of the policy deck is often the most important and most revealing about a players action. It is only during this point in the game that all players know the number of liberal cards in the deck. From this we can work out the odds of drawing at least one liberal card at these stages. After governments have passed this becomes harder due to the possibilities that players have lied. 
+The first government of the policy deck is often the most important and most revealing about a player's action. It is only during this point in the game that all players know the number of liberal cards in the deck. From this we can work out the odds of drawing at least one liberal card at these stages. After governments have passed this becomes harder due to the possibilities that players have lied. 
```

Comment:
```suggestion
The first government of the policy deck is often the most important and most revealing about a players' action. It is only during this point in the game that all players know the number of liberal cards in the deck. From this we can work out the odds of drawing at least one liberal card at these stages. After governments have passed this becomes harder due to the possibilities that players have lied. 
```

Reply:
Do you want to pluralize "player", like, "The first government of the policy deck is often the most important and most revealing about players' actions."? If we're talking about one player, the apostrophe before the S is correct.

Reply:
Yea it's just a single player so your change is right. Had to revisit my understanding of grammar.